noir-projects/aztec-nr/aztec/src/macros/internals_functions_generation/abi_marker_attributes.nr

```diff
@@ -0,0 +1,27 @@
+//! Marker attributes that are applied by the Aztec.nr macros to the generated functions with the
+//! `__aztec_nr_internals___` prefix in the name. The attributes are prefixed with `abi_` as they are used only for ABI
+//! purposes and artifact generation.
```

Comment:
I like this. Might just call them attributes though.

Reply:
Dropped "marker" 👍